### DISCUSSION TITLE
Recognition of DAOs as legal entities?

### DISCUSSION THREAD

**Discussion by jonathancross** [liberland/constitution#591] [Ideas]
I am curious if there has been discussion on recognition of something like a DAO as a legal entity?
AFAIK, under current international law, a (truly decentralized) DAO cannot legally own anything, open a bank account, pay taxes, etc.

It would be interesting if Liberland became the first country to offer DAOs official legal status in clear language.

There are some [hacks and workarounds in a few jurisdictions](https://nomoretax.eu/legal-aspects-of-decentralized-autonomous-organisations-daos/).

**Comment by sergiofspedro** [comment]
Hello Jonathan,
that is an ongoing issue for countries and regional blocks. E.g. [Marshall Islands already recognized DAOs](https://coingeek.com/marshall-islands-declares-daos-as-legal-entities-in-pioneering-move/), as well as Vermont, Wyoming and Tennessee, and the [EU is starting that discussion now](https://www.coindesk.com/policy/2023/01/17/finnish-minister-calls-for-eu-law-to-recognize-daos/).

In my perspective, I see potential gains and pitfalls with the recognition as it will create relative easiness of DAOs with the bureaucracies of existing public institutions, but it can also be a tool to create a framework of control and rigidity to DAOs and their work.

I would risk saying that a legal framework can possibly exist in the future, but it would depend on the will of the DAO in concrete to adhere to that statute or not.

FYI:
[Aragon´s take on this topic](https://aragon.org/how-to/choose-a-legal-wrapper-for-your-dao)
[DAOcolective collection of articles on the topic](https://daocollective.xyz/advanced-search/?_p04674766=legal)
[Harvard legal scholar opinion](https://corpgov.law.harvard.edu/2022/09/17/a-primer-on-daos/)
